stacks/epinio/common.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
```

Comment:
Using a shared file is complicated, especially if the script should be runnable from any working dir.

How many of these functions and variables are really shared between install/uninstall/upgrade?
This adds a lot of global state to the individual scripts.